packages/contracts/tasks/validate-address-dictator.ts

```diff
@@ -0,0 +1,141 @@
+'use strict'
+
+import { ethers } from 'ethers'
+import { task } from 'hardhat/config'
+import * as types from 'hardhat/internal/core/params/argumentTypes'
+import { hexStringEquals } from '@eth-optimism/core-utils'
+import { getContractFactory } from '../src/contract-defs'
+
+import { getInput, color as c } from '../src/task-utils'
+
+const printComparison = (
+  action: string,
+  description: string,
+  value1: string,
+  value2: string
+) => {
+  console.log(action + ':')
+  if (hexStringEquals(value1, value2)) {
+    console.log(c.green(`${description} looks good! 😎`))
+  } else {
+    throw new Error(`${description} looks wrong`)
+  }
+  console.log() // Add some whitespace
+}
+
+task('validate:address-dictator')
+  // Provided by the signature Requestor
+  .addParam(
+    'dictator',
+    'Address of the AddressDictator to validate.',
+    undefined,
+    types.string
+  )
+  // I'm not certain if this value should be entered manually or read from the artifacts,
```

Comment:
TODO reminder to finish this comment. I'm a proponent of entering manually, will non-deployers even have the artifacts yet?

Reply:
No. I removed that comment.